apps/studio/components/layouts/ProjectLayout/ProjectLayout.tsx

```diff
@@ -201,6 +205,10 @@ const ContentWrapper = ({ isLoading, isBlocking = true, children }: ContentWrapp
     selectedProject?.status === PROJECT_STATUS.PAUSING
   const isProjectOffline = selectedProject?.postgrestStatus === 'OFFLINE'
 
+  useEffect(() => {
+    if (ref) state.setSelectedDatabaseId(ref)
+  }, [ref])
```

Comment:
Is this still needed now that it's in a context?

Reply:
yup this is to initialize the value in the valtio store - unless there's a better way we can do this?